dqc_us_rules/dqc_us_0014.py

```diff
@@ -0,0 +1,91 @@
+# (c) Copyright 2016, XBRL US Inc. All rights reserved.
+# See license.md for license information.
+# See PatentNotice.md for patent infringement notice.
+import os
+
+from .util import facts, messages, neg_num
```

Comment:
We are unable to use full paths in DQC because of the way plugins are managed in arelle.